### ISSUE TITLE
Line breaks and Latex rendering

### ISSUE THREAD

**Issue by ringosham** [noties/Markwon#149]
* **Markwon version**: 4.0.1
* **Markwon Latex plugin**: 4.0.1

Thanks for the fast bug fixes. Here's another bug I found.

1. Please specify expected/actual behavior
This formula gets rendered twice. It's kind of random and it doesn't happen on every expression.
I speculate it's something when superscript, subscript and fractions are present in an expression? I might be wrong. I'm not sure.
Edit: The same issue appears in v3.x.x
![Screenshot_20190703-135158](https://user-images.githubusercontent.com/7325740/60593164-12f3c200-9d9a-11e9-87d9-71366c378cd4.jpg)

2. Please specify conditions/steps to reproduce (layout, code, markdown used, etc)

Activity class
```java
@Override
protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);
    TextView text = findViewById(R.id.test);
    Markwon markdown = Markwon.builder(this).usePlugin(JLatexMathPlugin.create(50)).build();
    markdown.setMarkdown(text, readMarkdownFile(R.raw.markdown));
}

public String readMarkdownFile(int resourceId) {
    InputStream in = getResources().openRawResource(resourceId);
    Scanner scan = new Scanner(in).useDelimiter("\\A");
    return scan.hasNext() ? scan.next() : "";
}
```

The markdown used.
```
$$ 
    \sigma_T^2 = \frac{1-p}{p^2}
$$
```

**Comment by noties** [comment]
Hello @ringosham !

Thank you for your feedback, it's really valuable! 👍 

About the issue: it was an interesting debugging session. Turns out that Android will draw a `drawable` for each line of text (for which `span` is applied). Yeah. And this is not related to actual formula, you can change `\sigma_T^2 = \frac{1-p}{p^2}` for any other and the result will still be duplicated. But if you remove the space character after opening `$$` when formula won't be duplicated. I'm going to change what text is actually added to a `TextView`, so formula won't have multiple lines (to be shown as a placeholder text). I'm going to publish `4.0.2-SNAPSHOT` soon.

Thanks again for your scrupulous bug finding! 🙌

**Comment by ringosham** [comment]
The snapshot seems to be working fine.

**Comment by noties** [comment]
Released in `4.0.2` 👍
